recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+c_compiler_version:    # [linux]
+- '9'                  # [linux and not aarch64]
```

Comment:
Have you tried the latest gcc, but with using the options recommended here?
https://github.com/DLTcollab/sse2neon/tree/master#usage

Reply:
yes, I did try with latests gcc, and yes the test Makefile already have the compiler options sepcified:
https://github.com/DLTcollab/sse2neon/blob/31cb30b1f93256d7aa19bfbaf4558c1d35e9b3ba/Makefile#L33

Reply:
Have you tried outside of docker?